GNU parallel 20180722 & 64-bit-only

Mno-hime opened the pull request:
Upstream changelog (https://gitlab.com/GNU/Parallel/blob/master/NEWS)
says: "Better CPU detection on OpenIndiana".

Tested several examples from GNU parallel book.

AndWac reviewed (commented):
Can you explain why and what you changed tools/.gnupg/pubring.gpg?

Mno-hime committed:
GNU parallel 20180722 & 64-bit-only

Upstream changelog (https://gitlab.com/GNU/Parallel/blob/master/NEWS)
says: "Better CPU detection on OpenIndiana".

Tested several examples from GNU parallel book.

Mno-hime commented:
@AndWac The build system adds GPG signatures (or something of the sort) there when new `$(COMPONENT_SIG_URL)` is retrieved. But as it's binary file, Git is unable to resolve conflicts when other PR updates the file, and, hence, PRs gets in conflict. I pulled out the change.

Btw, you can view content of the file by `gpg2 --no-default-keyring --keyring pubring.gpg --list-keys`.
